src/script/microbit-interfacing/MicrobitBluetooth.ts

```diff
@@ -486,17 +510,31 @@ export const startBluetoothConnection = async (
 
 const requestDevice = async (name: string): Promise<BluetoothDevice | undefined> => {
   try {
-    return navigator.bluetooth.requestDevice({
-      filters: [{ namePrefix: `BBC micro:bit [${name}]` }],
-      optionalServices: [
-        MBSpecs.Services.UART_SERVICE,
-        MBSpecs.Services.ACCEL_SERVICE,
-        MBSpecs.Services.DEVICE_INFO_SERVICE,
-        MBSpecs.Services.LED_SERVICE,
-        MBSpecs.Services.IO_SERVICE,
-        MBSpecs.Services.BUTTON_SERVICE,
-      ],
-    });
+    const result = await Promise.race([
```

Comment:
```suggestion
    // In some situations the Chrome device prompt simply doesn't appear so we time this out after 30 seconds and reload the page
    const result = await Promise.race([
```

@microbit-robert I'm going to go with this comment for now but if you can narrow it down / reference a Chrome bug etc then please do.